### PR TITLE
Development

### PR DESCRIPTION
**Pull request by keshav-c**
Awaiting your comments. We haven't done much for the header and footer sections as per the project requirements.

We have created 2 pages, one representing the original and the other it's grey-scale heatmap as per our perception.

Both are linked together via hyper links in the header.

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+    text-align: center;
+    text-transform: capitalize;
+    top: 0;
+    width: 400px;
+}
+
+.featured-articles {
+    display: grid;
+    grid-auto-rows: 45em;
+    grid-template-columns: repeat(2, 1fr);
```
Comment (efrapp) [inline styles/main.css:48]: Don't worry about this error, it is something related to the [css lint](http://csslint.net/) Stickler uses. You can check a discussion about that here: https://github.com/CSSLint/parser-lib/issues/228

File: styles/main.css [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+    grid-template-rows: 5fr 4fr;
```
Comment (efrapp) [inline styles/main.css]: Same thing here and in all parts where this error appears.

**Review by efrapp — CHANGES_REQUESTED** [review]
# Good Job! but I want you to address some things

The purpose of this project is to enforce design principles so you made a great job creating the grayscale heat map page and also the other page was great but is not needed for this project, you don't need to delete it it is just to put the focus in the grayscale page because there you are applying the key concepts for this project.

The things I want you to address are the followings:

1. Apply the correction shown in the HTML validator [here](https://validator.w3.org/nu/?doc=https%3A%2F%2Fkeshav-c.github.io%2Fsmashing-mag%2Findex.html). This is for the `index.html` file but it is important that you have those corrections in mind.
2. Apply the corrections shown in the HTML validator for the [`heatmap.html`](https://validator.w3.org/nu/?doc=https%3A%2F%2Fkeshav-c.github.io%2Fsmashing-mag%2Fheatmap.html) file.
3. Please update your GitHub Page link because in the CSS validator it shows errors that you already solved. See [here](http://jigsaw.w3.org/css-validator/validator?uri=https%3A%2F%2Fkeshav-c.github.io%2Fsmashing-mag%2Fstyles%2Fmain.css).

For point 1 and 2 pay more attention to the errors than the warnings.

When you are done with this changes, please, submit a new code review to check the changes.

File: styles/main.css (hunk 1 above)
Comment (keshav-c) [reply styles/main.css:48]: Thanks. I guess w3 validators are going to be more useful from now on, if stickler can't even validate grid layouts.

**Comment by keshav-c** [comment]
I have made all corrections as suggested. Awaiting your re-review.

I have made the heatmap show up as index, so that is the focus and the original links from the heatmap page.

Also the warnings ask me to include section headings that are not a part of the original markup so I ended up ignoring those.

**Review by efrapp — APPROVED** [review]
# Ok you made a great job! 👍 

You can continue in your path to achieving the mastery!

You can submit now your completion form for this project.

I whish you the best.

**Pull request merged**